cluster/store/store.go

```diff
@@ -173,15 +173,20 @@ type Store struct {
 	mutex      sync.Mutex
 	candidates map[string]string
 
-	// lastAppliedIndexOnStart represents the index of the last applied command when the store is opened.
-	lastAppliedIndexOnStart uint64
+	// initialLastAppliedIndex represents the index of the last applied command when the store is opened.
+	initialLastAppliedIndex uint64
```

Comment:
**suggestion:**  do we want to bring it back to `initialLastAppliedIndex` ,  i thought `lastAppliedIndexOnStart` is more descriptive